packages/cli/src/NodeTypes.ts

```diff
@@ -8,12 +8,19 @@ import type {
 } from 'n8n-workflow';
 import { NodeHelpers } from 'n8n-workflow';
 import { Service } from 'typedi';
-import { RESPONSE_ERROR_MESSAGES } from './constants';
 import { LoadNodesAndCredentials } from './LoadNodesAndCredentials';
 import { join, dirname } from 'path';
 import { readdir } from 'fs/promises';
 import type { Dirent } from 'fs';
 
+class UnrecognizedNodeError extends Error {
+	severity = 'warning';
+
+	constructor(nodeType: string) {
+		super(`Node not recognized: ${nodeType}".`);
```

Comment:
Since the error message has changed, [this e2e test needs to be updated](https://github.com/n8n-io/n8n/blob/master/cypress/e2e/12-canvas.cy.ts#L393)